### PR TITLE
Add basic kernel tests

### PR DESCRIPTION
**Pull request by NeQuissimus**
This adds some very basic kernel tests.
Basically just starts a VM and makes sure the kernel has the expected version (and thereby making sure it could be built and starts up).

### CODE REVIEW

File: nixos/tests/kernel-lts.nix [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import ./make-test.nix ({ pkgs, ...} : {
+  name = "kernel-latest";
```
Comment (kamilchm) [inline nixos/tests/kernel-lts.nix]: `kernel-lts`?

**Comment by Mic92** [comment]
should we also backport this to `17.09`?

File: nixos/release-combined.nix [hunk 2]
```diff
@@ -91,6 +91,9 @@ in rec {
         (all nixos.tests.keymap.neo)
         (all nixos.tests.keymap.qwertz)
         (all nixos.tests.plasma5)
+        (all nixos.tests.kernel-copperhead)
```
Comment (fpletz) [inline nixos/release-combined.nix]: The copperhead kernel is in no way supported or release-critical so it shouldn't be part of the tested job.

Comment (NeQuissimus) [reply nixos/release-combined.nix]: I will take it back out

Comment (NeQuissimus) [reply nixos/release-combined.nix]: Fixed in 52341c03e80a594f49ddeaf86e78da6e834c6bb7

**Pull request merged**